ui/apps/platform/src/Containers/PolicyCategories/PolicyCategoriesPage.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import {
+    PageSection,
+    Bullseye,
+    Spinner,
+    Alert,
+    Divider,
+    Button,
+    Flex,
+    Toolbar,
+    ToolbarContent,
+    ToolbarItem,
+} from '@patternfly/react-core';
+
+import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
+import { getPolicyCategories } from 'services/PoliciesService';
+import PolicyManagementHeader from 'Containers/PolicyManagement/PolicyManagementHeader';
+import PolicyCategoriesListSection from './PolicyCategoriesListSection';
+
+function PolicyCategoriesPage(): React.ReactElement {
+    const [isLoading, setIsLoading] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
+    // TODO switch type once new API is in
+    const [policyCategories, setPolicyCategories] = useState<string[]>([]);
+
+    let pageContent = (
+        <PageSection variant="light" isFilled id="policies-table-loading">
+            <Bullseye>
+                <Spinner isSVG />
+            </Bullseye>
+        </PageSection>
+    );
+
+    if (errorMessage) {
+        pageContent = (
+            <PageSection variant="light" isFilled id="policies-table-error">
+                <Bullseye>
+                    <Alert variant="danger" title={errorMessage} />
+                </Bullseye>
+            </PageSection>
+        );
+    }
+
+    if (!isLoading && !errorMessage && policyCategories.length > 0) {
+        pageContent = <PolicyCategoriesListSection policyCategories={policyCategories} />;
+    }
+
+    useEffect(() => {
+        setIsLoading(true);
+        getPolicyCategories()
+            .then((categories) => {
+                setPolicyCategories(categories);
+                setErrorMessage('');
+            })
+            .catch((error) => {
+                setPolicyCategories([]);
+                setErrorMessage(getAxiosErrorMessage(error));
+            })
+            .finally(() => setIsLoading(false));
+    }, []);
```

Comment:
Not a blocker to merge, it seems like `useEffect` is often closer to `useState` hooks and `pageContent` is closer to `return` statement.